app/bingey-api/models/user.js

```diff
@@ -0,0 +1,52 @@
+const { Schema, model } = require('mongoose');
+
+const userSchema = new Schema({
+    _id:{
```

Comment:
User should have autogenerated IDs, the _Title_ had manually added ones cause we will use the ones from IMDb